deployment/helm/eclipse-ditto/templates/concierge-deployment.yaml

```diff
@@ -47,9 +47,8 @@ spec:
         - name: {{ .Chart.Name }}-concierge
           image: "{{ .Values.concierge.image.repository }}:{{ .Values.concierge.image.tag }}"
           imagePullPolicy: {{ .Values.concierge.image.imagePullPolicy }}
-          command: ["/sbin/tini"]
+          command: ["java"]
```

Comment:
Unfortunately this will override the Docker entrypoint. Property naming is misleading (see https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes)
`java` should be moved to `args` and `command` field can be removed.

Reply:
Thx for the hint - now it should finally be correct :D